File: app/src/main/java/com/example/myapplication/database/Card.java
package com.example.myapplication.database;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.io.Serializable;

@Entity
public class Card implements Serializable {
    @PrimaryKey
    public int id;

    public int presetId;
    public String name;
    public String gender;
    public String fio;
}

File: app/src/main/java/com/example/myapplication/Card.kt
package com.example.myapplication

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class Card(
    @SerializedName("preset") val preset: Preset,
    @SerializedName("name") var name: String,
    @SerializedName("gender") var gender: String,
    @SerializedName("fio") var fio: String
) : Serializable {
    @SerializedName("imagePath")
    lateinit var imagePath: String

    @SerializedName("pdfPath")
    lateinit var pdfPath: String

    // TODO: get image and pdf from api and save locally
}